trtools/prancSTR/README.rst

```diff
@@ -0,0 +1,111 @@
+.. overview_directive
+.. |prancSTR overview| replace:: prancSTR takes in a VCF file generated by HipSTR from one or more samdples an identifies STRs with evidence of somatic mosaicism.
+.. overview_directive_done
+
+prancSTR
+=========
+
+Population-level heterogeneity generally arises due to germline mutations that occur before the formation of the zygote and are inherited by all cells in the offspring.
+However, heterogeneity within an individual may also exist due to somatic mutations that occur post-zygotically in only a sub-population of cells (). 
+
+Short tandem repeats (STRs) consist of repeated sequences of 1-6bp and comprise more than 1 million loci in the human genome. Somatic mosaicism at STRs is known 
+to play a key role in the pathogenicity of loci implicated in repeat expansion disorders, and is highly prevalent in cancers exhibiting microsatellite instability.
+While a variety of tools have been developed to genotype germline variation at STRs, no method currently exists for systematically identifying mosaic STRs (mSTRs).
+
+We introduce prancSTR, a novel method for detecting mSTRs from individual next-generation sequencing datasets. Unlike many existing mosaicism detection methods 
+for other variant types, prancSTR does not require a matched control sample as input.
+
+prancSTR is in *beta* version while the manuscript is in preparation
+
+
+Usage
+-----
+To run prancSTR use the following command::
+
+	prancSTR \
+  	  --vcf <vcf file> \
+  	  --out <string> \
+  	  [filter options]
+
+Required parameters:
+
+* :code:`--vcf <VCF>` Input VCF file, generated by HipSTR. 
+* :code:`--out <string>` Output file prefix. Use :code:`stdout` to print file to standard output
+
+prancSTR will output a tab-delimited file with candidate mosaic STRs (mSTRs) either to stdout or to :code:`$out.tab`. See a description of the output file below.
+
+Other general parameters:
+
+* :code:`--region <string>`: Restrict to the region chr:start-end. VCF file must be bgzipped and indexed to use this option.
+* :code:`--samples <string>`: Restrict to the given list of samples. Samples are comma separated
+* :code:`--vcftype <string>`: Specify the tool which generated the vcf call file for STRs. e.g. hipstr, gangstr etc.
+* :code:`--only-passing <action=store_true>`: Filter out segmental duplication regions.
+* :code:`--output-all <action=store_true>`: Force tool to output results for all loci.
+* :code:`--debug <action=store_true>`: Print helpful debug messages.
+* :code:`--quiet <action=store_true>`: Restrict printing of any messages.
+* :code:`--version`: Print the version of the tool
+
+See `Example Commands`_ for examples running prancSTR under different settings.
+
+prancSTR works downstream of HipSTR (or any other genotyper that provides estimated diploid repeat lengths and the observed distribution of copy numbers across all reads aligning to a locus). 
```

Comment:
It can work downstream of any genotyper that produces output in a similar format as HipSTR. All it needs is a vcf file with as said estimated diploid repeat lengths and observed distribution of copy numbers. And any other TR caller whose output can spit this, should also work with prancSTR.

Reply:
As far as I know, there are no other genotypers which match HipSTR's output - they all have their own output formats. Unless you know of a genotyper that could be used with prancSTR without additional jury-rigging by the user, I would not have this hypothetical mentioned in user-facing docs.

In the portion of the README which says "these are the fields we use", you could specify exactly what the field names are. Users that want to do jury-rigging could figure things out from there without needing prompting like this.

Reply:
That makes complete sense and I will work on rewording the README to account for this.